tests/test-sources/plugins/snippets/nvim-snippets.nix

```diff
@@ -0,0 +1,93 @@
+{
+  empty = {
+    plugins = {
+      cmp.enable = true;
+      nvim-snippets.enable = true;
+    };
+  };
+
+  settings = {
```

Comment:
Usually we have an `defaults` test case that strictly copies the upstream defaults and eventually an `example` test case where we set non-defaults value.